homeassistant/components/miraie_ac/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "miraie_ac",
+  "name": "MirAIe AC",
+  "codeowners": ["@milothomas"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/miraie_ac",
+  "iot_class": "cloud_push",
+  "requirements": ["py-miraie-ac==0.1.10"]
```

Comment:
On https://github.com/milothomas/py-miraie-ac/blob/main/setup.cfg the version is still `0.1.9` therefore I can't make the sanity check for the specified version. Please update your GitHub repo to include the specified version